ols/app/models/models.py

```diff
@@ -1,24 +1,25 @@
 """Data models representing payloads for REST API calls."""
 
 from typing import Union
+from uuid import UUID, uuid4
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 
 class LLMRequest(BaseModel):
     """Model representing a request for the LLM (Language Model).
 
     Attributes:
         query: The query string.
-        conversation_id: The optional conversation ID.
+        conversation_id: The optional conversation id in valid UUID format.
         response: The optional response.
 
     Example:
         llm_request = LLMRequest(query="Tell me about Kubernetes")
     """
 
     query: str
-    conversation_id: Union[str, None] = None
+    conversation_id: UUID = Field(default_factory=uuid4)
```

Comment:
does this change break the existing gradio UI?

also if it doesn't break any existing tests that means we are missing tests that should be added a part of this change to enforce the api.

Reply:
Some tests are failing already. I wanted to first ensure that this change makes sense to you. If so, I'll update tests and cover missing bits.

Reply:
UI is not broken with this change

Reply:
And I file a bug about missing conv cache and do it as a part of OLS-40, ok?

Reply:
> And I file a bug about missing conv cache and do it as a part of OLS-40, ok?

incorporation the conversation cache into all the flows is part of https://issues.redhat.com/browse/OLS-31 and in particular https://issues.redhat.com/browse/OLS-59